### PR TITLE
fix: WebAssembly build for core packages

### PR DESCRIPTION
**Pull request by xxuejie**

### CODE REVIEW

**Comment by zhangsoledad** [comment]
bors r=driftluo,zhangsoledad

File: Makefile [hunk 1]
```diff
@@ -17,6 +17,10 @@ test: ## Run all tests.
 cov: ## Run code coverage.
 	RUSTC="$$(pwd)/devtools/cov/rustc-proptest-fix" taskset -c 0 cargo tarpaulin --timeout 300 --exclude-files "*/generated/" "test/*" "*/tests/" --all -v --out Xml
 
+.PHONY: wasm-build-test
+wasm-build-test: ## Build core packages for wasm target
+	cd wasm-build-test && cargo build --target=wasm32-unknown-unknown
```
Comment (u2) [inline Makefile]: where is wasm-build-test?

**Comment by xxuejie** [comment]
bors r=driftluo,zhangsoledad,u2

**Comment by u2** [comment]
bors r=driftluo,zhangsoledad,u2

**Pull request merged**